src/main/java/ch/cern/eam/wshub/core/services/workorders/entities/EntitySafetyWSHub.java

```diff
@@ -0,0 +1,114 @@
+package ch.cern.eam.wshub.core.services.workorders.entities;
+
+import ch.cern.eam.wshub.core.annotations.InforField;
+import net.datastream.schemas.mp_fields.USERDEFINEDCODEID_Type;
+
+import java.math.BigDecimal;
+
+public class EntitySafetyWSHub {
```

Comment:
Name EntitySafetyWSHub might be confusing

Reply:
Changed to EntitySafety